spec/helpers/data_source_helper.rb

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2010-2019 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+module GoodData
```

Comment:
Why is this a helper instead of a proper Model class?

Reply:
I don't expect users to create Snowflake data sources using the SDK. Therefore I've chosen not to spend time on it.

Reply:
I feel like it would be the same amount of work like you did here, with the benefit of working the same as the other API endpoints and also adhering to the same object model instead of creating a "helper", which does not really denote any specific role in the model.

Reply:
Think about testing and documentation in addition to implementation. It would kill the rest of my day. That's a high price for something no one's going to use.